lib/views/pages/login_signup/login_page.dart

```diff
@@ -553,4 +560,27 @@ class _LoginScreenState extends State<LoginPage> with TickerProviderStateMixin {
           ),
         ));
   }
+
+  Future<void> checkAndSetUrl() async {
+    setState(() {
+      isUrlCalled = true;
+    });
+
+    try {
+      await http.get('${dropdownValue.toLowerCase()}://${urlController.text}/');
+
+      setAPIURL();
+      _setURL();
+    } catch (e) {
+      _scaffoldkey.currentState.showSnackBar(
+        SnackBar(
+            behavior: SnackBarBehavior.floating,
+            content: Text('Url does not exist')),
```

Comment:
Update message "Incorrect organization entered" or something more meaningful.
Make snackbar look good to go with current UI/UX. Currently snackbar looking very simple.

Reply:
Sure working on it